app/models/programme.rb

```diff
@@ -147,4 +149,26 @@ def snomed_procedure_code
   def snomed_procedure_term
     SNOMED_PROCEDURE_TERMS.fetch(type)
   end
+
+  SNOMED_TARGET_DISEASE_CODES = {
+    "hpv" => "240532009",
+    "flu" => "6142004"
+  }.freeze
+
+  def snomed_target_disease_code
+    SNOMED_TARGET_DISEASE_CODES.fetch(type)
+  end
+
+  SNOMED_TARGET_DISEASE_TERMS = {
+    "hpv" => "Human papillomavirus infection (disorder)",
+    "flu" => "Influenza (disorder)"
```

Comment:
Should use `Influenza` which is the preferred synonym for [6142004](https://termbrowser.nhs.uk/?perspective=full&conceptId1=6142004&edition=uk-edition&release=v20250604&server=https://termbrowser.nhs.uk/sct-browser-api/snomed&langRefset=999001261000000100,999000691000001104).